service/application/src/main/resources/application.yml

```diff
@@ -4,393 +4,75 @@ springdoc:
-      max-file-size: 10MB
-      max-history: 7
-      total-size-cap: 1GB
-
-# micrometer / statsd
-management:
-  metrics:
-    tags:
-      # key - value pairs
-      maas_id: my-maas-id
-    enable:
-      # enable / disable specific metrics
-      all: false
-      # sample metrics to be exposed
-      application:
-        started:
-          time: true
-      jvm:
-        info: true
-  statsd:
-    metrics:
-      export:
-        # enable / disable shipping metrics to StatsD endpoint
-        enabled: false
-        flavor: datadog
-        host: 127.0.0.1
-        port: 8125
-        protocol: udp
-
 server:
   port: 8180
-
-idGenerator:
-  originId: event_management_agent_${EP_EVENT_MANAGEMENT_AGENT_ID}
-
 spring:
-  datasource:
-    url: jdbc:h2:file:./data/cache;DB_CLOSE_ON_EXIT=FALSE
-    username: sa
-    password: password
-    driver-class-name: org.h2.Driver
-
-  jpa:
-    database-platform: org.hibernate.dialect.H2Dialect
-    hibernate:
-      ddl-auto: create-drop
-    defer-datasource-initialization: true
   h2:
     console:
       path: /h2
       enabled: true
       settings:
         web-allow-others: true
+  datasource:
+    password: password
+    driver-class-name: org.h2.Driver
+    username: sa
+    url: jdbc:h2:file:./data/cache;DB_CLOSE_ON_EXIT=FALSE
+  jpa:
+    defer-datasource-initialization: true
+    hibernate:
+      ddl-auto: create-drop
+    database-platform: org.hibernate.dialect.H2Dialect
   servlet:
     multipart:
       max-request-size: ${MAX_REQUEST_SIZE:5MB}
       max-file-size: ${MAX_FILE_SIZE:5MB}
   main:
     allow-bean-definition-overriding: true
-
 camel:
   springboot:
     use-mdc-logging: true
-  errorHandling:
-    maximumRedeliveries: 10
-    maximumRedeliveryDelay: 60000
-    redeliveryDelay: 1000
-
 kafka:
   client:
     config:
-      connections:
-        timeout:
-          value: 60_000
+      reconnections:
+        max-backoff:
+          unit: milliseconds
+          value: 1000
+        backoff:
           unit: milliseconds
+          value: 50
+      connections:
         max-idle:
-          value: 10_000
           unit: milliseconds
+          value: 10000
         request-timeout:
-          value: 5_000
-          unit: milliseconds
-      reconnections:
-        backoff:
-          value: 50
           unit: milliseconds
-        max-backoff:
-          value: 1_000
+          value: 5000
+        timeout:
           unit: milliseconds
-
+          value: 60000
 eventPortal:
-  runtimeAgentId: ${EP_RUNTIME_AGENT_ID:defaultAgentId}
-  organizationId: ${EP_ORGANIZATION_ID:defaultOrgId}
-  topicPrefix: ${EP_TOPIC_PREFIX:sc/ep/runtime}
+  organizationId: ${EP_ORGANIZATION_ID:localrcorg}
+  runtimeAgentId: ${EP_RUNTIME_AGENT_ID:3jczd7fk0xv}
+  incomingRequestQueueName: "ep_core_ema_requests_fakedcid"
+  managed: true
   gateway:
-    id: gateway
-    name: evmr1
+    id: f0ydyzsr1jp
+    name: POCCloudManagedEMAEVMR
```

Comment:
Please undo these changes.

Reply:
Done. See slack message how I mitigated the impact